webdis.json

```diff
@@ -26,6 +26,6 @@
 		}
 	],
 
-	"verbosity": 6,
+	"verbosity": 4,
```

Comment:
hmm… what if people upgrade and keep their previous `webdis.json`? Then they'll start logging a ton of stuff at TRACE.

Reply:
I changed it to 8, this is higher than in previous JSON files.